### ISSUE TITLE
为什么训练过程loss没有下降和不收敛的现象，而且loss很大，有没有人也是这样的

### ISSUE THREAD

**Issue by jingenyan** [Tianxiaomo/pytorch-YOLOv4#109]
Message: 'Train step_240: loss : 35656.95703125,loss xy : 118.43473815917969,loss wh : 51.72434616088867,loss obj : 33237.66015625，loss cls : 2249.13525390625,loss l2 : 13105.2314453125,lr : 2.0735999999999997e-07'

**Comment by Derekabc** [comment]
same problem, any updates?

**Comment by Softwaring** [comment]
一样的问题，请问你解决了吗？ @jingenyan

**Comment by Nicole1130** [comment]
出现了同样的问题，希望得到解答 @Tianxiaomo

**Comment by architect-road** [comment]
same problem@Tianxiaomo

**Comment by admin221** [comment]
same problem

**Comment by wuyong139** [comment]
same problem

**Comment by hhhhhhogan** [comment]
same problem, any updates?

**Comment by jingtianyilong** [comment]
Same problem. Also notice that the final prediction results have extremly large box width. The center x,y and height are pretty much accurate. But you see generally large `loss_xy` but low `loss_wh`. I tried with yolov4.pth, no problem at all. I've checked the visualization, dataset/dataloader and forwarding part. Seems fine. So I assume that the loss term might be the cause. 
I was confused by the code in `yolo_loss`. Hope someone can help. Great appreciate. @Tianxiaomo

**Comment by jingtianyilong** [comment]
https://github.com/AlexeyAB/darknet/blob/9db0ed96621bcc8bd4aba27b0e9662c6dc33f011/src/yolo_layer.c#L344-L675
This is the original loss term. Would definately check these code and see if there's anything we can do to.
It seems that `total_loss = iou_loss + classification_loss`. I also see label smoothness and some "normalizer"-things. The loss term in this project might have problem. 
Pls anyone who confused by this also help to check this.

**Comment by Fan-SR** [comment]
同样的问题，请问有解决吗
Train step_320: loss : 64961.03125,loss xy : 19.951534271240234,loss wh : 22.7925968170166,loss obj : 64887.9765625，loss cls : 30.31162452697754,loss l2 : 23576.07421875,lr : 1.6e-11

**Comment by 0Error0Warning** [comment]
I met this problem too, have you solved it?

**Comment by Mxcarlet** [comment]
感觉好像是obj_mask的问题

**Comment by Mxcarlet** [comment]
感觉是obj_mask的问题

**Comment by xlwan1132** [comment]
learning_rate is so small
